src/components/Slices/SlicesTable.jsx

```diff
@@ -1,56 +1,114 @@
 import React from "react";
 import PropTypes from "prop-types";
 import MaterialTable from "material-table";
-import { Typography, Box } from "@material-ui/core";
+import { Typography, Box, makeStyles } from "@material-ui/core";
 import { satoshisToBitcoins } from "unchained-bitcoin";
 
+import {
+  compareSlicesByPath,
+  compareSlicesByBalance,
+  compareSlicesByUTXOCount,
+  compareSlicesByTime,
+} from "../../utils/slices";
```

Comment:
Added some utilities for different ways to compare slices. This doesn't need to be specific to components or reducers, so put it in utils. These are used for the custom sorting algorithm for the table columns.